File: Modul5/Soal4/PRAK404-2210817110008-Aditya Oktaviari.c
#include <stdio.h>

int main (void)
{
    int i;
    float a, b;
    while(i != 5){
        printf("Pilih program :\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan pilihan : ");
        scanf("%d", &i);
        if(i>5 || i<=0){
            printf("Input anda salah, silahkan coba lagi\n\n");
        }
        else if(i==5){
            break;
        }
        else{
            printf("Masukan Nilai Pertama : ");
            scanf("%f", &a);
            printf("Masukan Nilai Kedua : ");
            scanf("%f", &b);
            if(i==1){
                printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n",a,b,a+b);
            }
            else if(i==2){
                printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n\n",a,b,a-b);;
            }
            else if(i==3){
                printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n\n",a,b,a*b);
            }
            else if(i==4){
                printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n\n",a,b,a/b);
            }
        }
    }
    printf("Terimakasih telah menggunakan kalkulator Aditya Oktaviari");
}

File: Modul5/Soal4/PRAK404-2210817110008-Aditya Oktaviari.py
i=int()
while(i != 5):
        print("Pilih program :")
        print("1. Penjumlahan")
        print("2. Pengurangan")
        print("3. Perkalian")
        print("4. Pembagian")
        print("5. Exit")
        i=int(input("Masukkan pilihan : "))
        if(i>5 or i<=0):
            print("Input anda salah, silahkan coba lagi\n")
        
        elif(i==5):
            break
        
        else:
            a=float(input("Masukan Nilai Pertama : "))
            b=float(input("Masukan Nilai Kedua : "))
            if(i==1):
                print("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n" % (a,b,a+b))
    
            elif(i==2):
                print("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n" % (a,b,a-b))
            
            elif(i==3):
                print("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n" % (a,b,a*b))
            
            elif(i==4):
                print("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n" % (a,b,a/b))

print("Terimakasih telah menggunakan kalkulator Aditya Oktaviari")

